lib/DataObject/ClassBuilder/PHPClassWriterInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\DataObject\ClassBuilder;
+
+use Pimcore\Model\DataObject\ClassDefinition;
+
+interface PHPClassWriterInterface
+{
+    public function writePHPClasses(ClassDefinition $classDefinition): void;
```

Comment:
Note, Symfony typically uses `dump` for this type of role, it's a "dumper", not a "writer".

Reply:
good to know, I'll rename it

Reply:
done